lib/PlacesProvider.js

```diff
@@ -194,17 +194,27 @@ Links.prototype = {
       limit = HISTORY_RESULTS_LIMIT;
     }
 
-    let sqlQuery =  `SELECT moz_places.url as url, moz_favicons.data as favicon, mime_type as mimeType,
-                            title, frecency, last_visit_date as lastVisitDate,  "history"  as type
+    let sqlQuery =  `SELECT moz_places.url as url,
+                            moz_favicons.data as favicon,
+                            moz_favicons.mime_type as mimeType,
+                            moz_places.title,
+                            moz_places.frecency,
+                            moz_places.last_visit_date as lastVisitDate,
+                            "history"  as type,
+                            moz_bookmarks.guid as bookmarkGuid,
+                            moz_bookmarks.dateAdded / 1000 as bookmarkDateCreated
```

Comment:
why are we dividing by 1000 here?

Looking at my places DB, both `last_visit_date` and `dateAdded` seem to have the same unit of time for me.

```python
In [1]: places = 1454547050246805

In [2]: bookmark = 1327245403927000

In [3]: def num_digits(num):
   ...:     count = 0
   ...:     while num > 0:
   ...:         num /= 10
   ...:         count += 1
   ...:     return count
   ...:
In [4]: num_digits(places)
Out[4]: 16

In [5]: num_digits(bookmark)
Out[5]: 16
```